### PR TITLE
V1 metadata publish

### PR DESCRIPTION
**Pull request by tamas-molnar**
- brightcove-annotations-rw
- v1-content-annotator-brightcove
- v1-content-annotator-binding-service
- v1-suggestor
- v1-annotations-rw
- cms-metadata bridge

### CODE REVIEW

File: kube-config-files/brightcove-annotations-rw-red.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: brightcove-annotations-rw-red
+  labels:
+    app: brightcove-annotations-rw-red
+    visualize: "true"
+spec:
+  replicas: 2
```
Comment (MMoisa) [inline kube-config-files/brightcove-annotations-rw-red.yaml]: There is only one replica in the production cluster

File: kube-config-files/v1-annotations-rw-red-service.yaml [hunk 2]
```diff
@@ -0,0 +1,71 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: v1-annotations-rw-red
+  labels:
+    app: v1-annotations-rw-red
+    visualize: "true"
+spec:
+  replicas: 2
```
Comment (MMoisa) [inline kube-config-files/v1-annotations-rw-red-service.yaml]: There is only one replica in the production cluster

File: kube-config-files/v1-content-annotator-binding-service-red-service.yaml [hunk 3]
```diff
@@ -0,0 +1,77 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: v1-content-annotator-binding-service-red
+  labels:
+    app: v1-content-annotator-binding-service-red
+    visualize: "true"
+spec:
+  replicas: 2
```
Comment (MMoisa) [inline kube-config-files/v1-content-annotator-binding-service-red-service.yaml]: There is only one replica in the production cluster

File: kube-config-files/v1-content-annotator-brightcove-red-service.yaml [hunk 4]
```diff
@@ -0,0 +1,77 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: v1-content-annotator-brightcove-red-service
+  labels:
+    app: v1-content-annotator-brightcove-red-service
+    visualize: "true"
+spec:
+  replicas: 2
```
Comment (MMoisa) [inline kube-config-files/v1-content-annotator-brightcove-red-service.yaml]: There is only one replica in the production cluster

**Review by sbuliarca — APPROVED** [review]

**Pull request merged**